src/main/java/cubicchunks/asm/mixin/fixes/common/MixinBlockChorusFlower.java

```diff
@@ -0,0 +1,49 @@
+/*
+ *  This file is part of Cubic Chunks Mod, licensed under the MIT License (MIT).
+ *
+ *  Copyright (c) 2015 contributors
+ *
+ *  Permission is hereby granted, free of charge, to any person obtaining a copy
+ *  of this software and associated documentation files (the "Software"), to deal
+ *  in the Software without restriction, including without limitation the rights
+ *  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ *  copies of the Software, and to permit persons to whom the Software is
+ *  furnished to do so, subject to the following conditions:
+ *
+ *  The above copyright notice and this permission notice shall be included in
+ *  all copies or substantial portions of the Software.
+ *
+ *  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ *  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ *  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ *  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ *  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ *  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ *  THE SOFTWARE.
+ */
+package cubicchunks.asm.mixin.fixes.common;
+
+import cubicchunks.world.ICubicWorld;
+import net.minecraft.block.BlockChorusFlower;
+import net.minecraft.client.Minecraft;
+import net.minecraft.world.World;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.Constant;
+import org.spongepowered.asm.mixin.injection.ModifyConstant;
+
+
+/**
+ * Created by Babbaj on 1/25/2018.
+ */
+@Mixin(BlockChorusFlower.class)
+public class MixinBlockChorusFlower {
+
+    // Allows the chorus plant to grow above Y 256
+    @ModifyConstant(method = "updateTick",
+            constant = @Constant(intValue = 256))
+    private int updateTick(int maxY) {
+        World world = Minecraft.getMinecraft().world;
```

Comment:
`Minecraft.getMinecraft()` available only at Client side. At forge dedicated server it will throw exception. Instead, add a function parameters of a `updateTick` after `int maxY` and use them;
```
private int updateTick(int maxY, World worldIn, BlockPos pos, IBlockState state, Random rand) {
return ((ICubicWorld) worldIn).getMaxHeight();
}
```

Reply:
As said above you should use the world from method argument this only accidentally works on single player and it will occasionally crash because of concurrent world access.

Reply:
fixed